app/code/Magento/Newsletter/Model/Queue.php

```diff
@@ -199,10 +200,18 @@ public function isNew()
      *
      * @param string|null $startAt start date of the mailing queue
      * @return $this
+     * @throws InvalidArgumentException
      */
     public function setQueueStartAtByString($startAt)
     {
-        if ($startAt === null || $startAt == '') {
+        if (!(is_string($startAt) || $startAt === null)) {
```

Comment:
new exceptions are not allowed here. We may try to change logic to call `$this->setQueueStartAt($this->utcConverter->convertLocalizedDateToUtc($startAt))` when `$startAt` is not empty string and set null for all other cases.
If there is issue that comes from` $this->utcConverter->convertLocalizedDateToUtc($startAt)` which may return false we need to handle that two

Reply:
Hello @xmav 
Thank you for your review
The logic has been fixed, tests too.
Examples for possible conditions are provided [here](https://3v4l.org/cA5GX)